client/src/pages/Contact.js

```diff
@@ -0,0 +1,235 @@
+              <img src={circleIcon} alt='circle Icon' />
+              <h3>{selectedAnimalToContact.age} years old</h3>
+            </AnimalAge>
+          </AnimalInfo>
+        </AnimalCard>
+      </AnimalCardWrapper>
+      <TextWrapper>
+        <h3>
+          Are you interested in adopting{' '}
+          {selectedAnimalToContact.name.toUpperCase()}?
+        </h3>
+        <p>
+          At <span> PAWZIES</span> we want you to find and adopt your{' '}
+          <span>Friend For Life</span>. In order to achieve this, it is crucial
+          to ensure that you and your chosen animal are 100% compatible. Below,
+          we provied all the available contact information of{' '}
+          <span>{selectedOrganizationToContact.name}</span> so you can easily
+          get in touch with them to discuss availability and prerequisites or
+          schedule a meeting to get to know{' '}
+          <span>{selectedAnimalToContact.name}</span> better!
+        </p>
+      </TextWrapper>
+      <OrganizationWrapper>
+        <h3>Organization Name</h3>
+        <h2>{selectedOrganizationToContact.name.toUpperCase()}</h2>
+        <h3>Organization E-Mail</h3>
+        <span>{selectedOrganizationToContact.email}</span>
+        <h3>Organization Phone Number</h3>
+        <span>{selectedOrganizationToContact.phone}</span>
+        <h3>Organization Address</h3>
+        <AddressWrapper>
+          <span>{selectedOrganizationToContact.street}</span>
+          <span>{selectedOrganizationToContact.zip}</span>
+          <span>{selectedOrganizationToContact.city}</span>
+        </AddressWrapper>
+      </OrganizationWrapper>
+
+      <Navigation />
+    </ContactWrapper>
+  );
+}
+const ContactWrapper = styled.section`
+  display: grid;
+  place-items: center;
+
+  margin-bottom: 1.5rem;
+
+  h1 {
+    margin: 1rem 0 0 2rem;
+    text-align: center;
+  }
+  h3 {
+    text-align: center;
+  }
+`;
+
+const TextWrapper = styled.section`
+  display: grid;
+  place-items: center;
+
+  background-color: var(--secondary-dark);
+  border-radius: 2rem;
+  color: var(--gray);
+  filter: drop-shadow(0 0 0.2rem var(--black));
+  padding: 1rem;
+  width: 90vw;
+
+  h3 {
+    margin-bottom: 1rem;
+  }
+
+  p {
+    letter-spacing: 0.05rem;
+
+    span {
+      font-weight: bold;
+      color: var(--white);
+    }
+  }
+`;
+
+const AnimalCardWrapper = styled.section`
+  display: grid;
+  place-items: center;
+
+  margin-bottom: 1rem;
+`;
+
+const AnimalCard = styled.section`
+  display: flex;
+  align-items: center;
+  flex-flow: column wrap;
+  justify-content: center;
+
+  background-color: var(--white);
+  border-radius: 2rem;
+  filter: drop-shadow(0 0 0.2rem var(--black));
+  letter-spacing: 0.1rem;
+  margin: 1rem 0;
+  padding: 0.5rem 0;
+  width: 90vw;
+`;
+
+const AnimalProfilePicture = styled.img`
+  max-height: 27rem;
+  max-width: 90vw;
+`;
+
+const AnimalName = styled.h2`
+  color: var(--secondary);
+  font-family: var(--ff-cursive);
+  font-size: 2rem;
+
```

Comment:
why this empty space?

Reply:
I have no idea :D I removed it, thank you!